docs/core/testing/unit-testing-mstest-configure.md

```diff
@@ -100,3 +100,97 @@ Each element of the file is optional because it has a default value.
 
 </RunSettings>
 ```
+
+## testconfig.json
+
+Starting with MSTest 3.7, when running your tests through MSTest runner (a.k.a Microsoft.Testing.Platform), you can use a `testconfig.json` file to configure the behavior of the test runner. The `testconfig.json` file is a JSON file that contains the configuration settings for the test runner. The file is used to configure the test runner and the test execution environment. For more information, refer to [Microsoft.Testing.Platform testconfig.json documentation](unit-testing-platform-config.md#testconfigjson).
```

Comment:
Does this article describe just the parts that are useful to configure MSTest, and the rest should be found in the other doc that is linked (but the link does not work and points to MD file, rather then a learn article)?

If yes, then it is not obvious from the description. Especially this part confuses me: 

file to configure the behavior of the test runner.  .... The file is used to configure the test runner ... for more info see the link. 

From that I don't see what this article is about, when for test runner you redirected me to the link.